root_VS2012/files/resource/Sql/sqlserver/AsyncProcessingService/AsyncProcessingServiceInsert.xml

```diff
@@ -1,7 +1,7 @@
-<?xml version="1.0" encoding="shift_jis" ?>
+﻿<?xml version="1.0" encoding="shift_jis" ?>
```

Comment:
I wrote this to [here](https://github.com/OpenTouryoProject/OpenTouryo/issues/117#issuecomment-161257146).
Please merge.